visualizer/index.js

```diff
@@ -140,6 +140,16 @@ function parseGrammar(source) {
     triggerRefresh(250);
   });
 
+  var actionContainers = document.querySelectorAll('.actionEntries');
```

Comment:
Could you put the following comment above this?

```
// TODO: Move this code into parseTree.js.
```

This code is really specific to the parseTree, but I see that you need it here to be able to call `triggerRefresh`. I think @mroeder's refactoring is going to make stuff like this easier.